core/src/main/java/org/elasticsearch/search/sort/ScriptSortBuilder.java

```diff
@@ -78,22 +146,189 @@ public ScriptSortBuilder setNestedPath(String nestedPath) {
         return this;
     }
 
+    /**
+     * Gets the nested path.
+     */
+    public String getNestedPath() {
```

Comment:
Nitpick: This mixes plain old POJO getters/setters with fluent interface like getters/setters. In another PR I left them untouched as well. Maybe at least track in a separate issue to unify those?